tests/integration/conftest.py

```diff
@@ -111,7 +113,7 @@ def mix_table1(engine, metadata):
     table_name = "mix_table1"
     columns = [
         sa.Column("col_int", sa.Integer()),
-        sa.Column("col_varchar", sa.String()),
+        sa.Column("col_varchar", sa.String(40)),

```

Comment:
I wouldn't have expected this to break `impala`'s integration tests.

Reply:
Haha I wasn't exactly hoping for this either. I'll try to figure something out... Worst case we need a case distinction.